examples/icd_code_prediction/config.yml

```diff
@@ -0,0 +1,11 @@
+Spacy:
+  processors: ["sentence", "tokenize", "pos", "ner", "umls_link"]
+  medical_onto_type: "ftx.medical.clinical_ontology.MedicalEntityMention"
+  umls_onto_type: "ftx.medical.clinical_ontology.UMLSConceptLink"
+  lang: "en_ner_bionlp13cg_md"
+
+Negation:
+  negation_rules_path: ""
```

Comment:
We can remove configs that's not being used anywhere.

Reply:
Thanks. I will fix those.